packages/node-sdk/src/client.ts

```diff
@@ -501,9 +496,10 @@ export class BucketClient {
    * The company must be set using `withCompany` before calling this method.
    * If the user is set, the company will be associated with the user.
    **/
-  public async updateUser(userId: string, opts?: TrackOptions) {
+  public async updateUser(userId: string | number, opts?: TrackOptions) {
     ok(
-      typeof userId === "string" && userId.length > 0,
+      (typeof userId === "string" && userId.length > 0) ||
+        typeof userId === "number",
```

Comment:
The error says "companyId" probably my copy paste mistake from the past or something